### DISCUSSION TITLE
[Feature] Location & rich text gallery support

### DISCUSSION THREAD

**Discussion by przybylski** [immich-app/immich#11698] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Whenever I create a gallery I would like to include some summary text, description of a certain location/photo or mark the trip on the map.

It would be great if gallery tool would support such entries, this would require custom ordering as well, so maps and text are aligned with photos

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3896
